src/Saas.Admin/readme.md

```diff
@@ -8,70 +8,74 @@ The SaaS Admin Service is an API that is reponsible for tenant management operat
 
 3. Saas.Admin.Service.Tests - Unit tests for the service
 
-## Project Overview
+## 1. Module Overview
 
-For more information on this module, please see the admin service page on our documentation site [here](https://azure.github.io/azure-saas/components/admin-service/).
+This project hosts a service api which serves as a gateway to administrate the SaaS ecosystem of Tenants. It is fully self-contained such that it includes complete copies of all necessary classes for operation. Since it contains no direct references to the other projects, it can be extracted to launch in isolation insofar as expected substitutes are configured.
 
-## How to Run Locally
+For a complete overview, please see the [SaaS.Admin.Service](https://azure.github.io/azure-saas/components/admin-service/) page in our documentation site.
 
-### Requirements
+## 2. How to Run Locally
+
+Once configured, this app presents an api service which manages authorization of actions issued to the Admin service. It may be run locally during development of service logic and for regenerating its included NSwag api client. (An NSwag file is included in the Admin project to generate its client.)
+
+### i. Requirements
 
 To run the web api, you must have the following installed on your machine:
 
 - [.NET 6.0](https://dotnet.microsoft.com/en-us/download/dotnet/6.0)
-
+- [ASP.NET Core 6.0](https://docs.microsoft.com/en-us/aspnet/core/introduction-to-aspnet-core?view=aspnetcore-6.0)
 - (Reccomended) [Visual Studio](https://visualstudio.microsoft.com/downloads/) or [Visual Studio Code](https://code.visualstudio.com/download)
-
-You must also have the following:
-
-- A connection string to a running, empty SQL Server Database
-  - Some options for using SQL Server Locally which we reccomend:
-    - [Local DB](https://docs.microsoft.com/en-us/sql/database-engine/configure-windows/sql-server-express-localdb?view=sql-server-ver15) (Windows Only)
+- A connection string to a running, empty SQL Server Database.
+    - [Local DB](https://docs.microsoft.com/en-us/sql/database-engine/configure-windows/sql-server-express-localdb?view=sql-server-ver15) (Windows Only) - See `Additional Resources` below for basic config secret
     - [SQL Server Docker Container](https://hub.docker.com/_/microsoft-mssql-server)
     - [SQL Server Developer Edition](https://www.microsoft.com/en-us/sql-server/sql-server-downloads)
+- A user store compatible with Microsoft Identity
```

Comment:
Change this to be something along the lines of "A deployed instane of the identity framework" and link to the quick start guide on step 1. https://azure.github.io/azure-saas/quick-start/